polar-core/tests/integration_tests.rs

```diff
@@ -414,6 +414,32 @@ fn test_jealous() -> TestResult {
     Ok(())
 }
 
+#[test]
+fn test_forall_with_dot_lookup() -> TestResult {
+    let p = polar();
+    p.load_str(
+        r#"
+        foo(x) if forall(y in x.ys, y != 0);
```

Comment:
What about w/ multiple terms on RHS? `y != 0 and y < 5`

Reply:
Sure, it's not really relevant to this test though -- internally that's still just one expression & it works exactly the same

Reply:
Oh okay, no worries then